changelog/november2023/2023-11-23-secret-manager-changed-secret-manager-console-pages-hav.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Secret Manager console pages have been updated
+status: changed
+author:
+    fullname: 'Join the #secret-manager channel on Slack.'
+    url: 'https://slack.scaleway.com/T7YEXCR7X/C04KGMME3U1'
+date: 2023-11-23
+category: security-identity
+product: secret-manager
+---
+
+In the Secret Manager section of the console, you can now select a region to display secrets and folders inside this region. Whenever a secret is created, there is no more need to specify the secret, as it will be created in the region you are currently working on.
```

Comment:
```suggestion
In the Secret Manager section of the [Scaleway console](https://console.scaleway.com), you can now select a [region](/identity-and-access-management/secret-manager/concepts/#region) to display secrets and folders inside this region. Whenever you create a secret, you no longer need to specify the region, as it will be created in the region you are currently working on.
```